product_docs/docs/pge/16/upgrading/major_upgrade.mdx

```diff
@@ -0,0 +1,155 @@
+    1. Start the database server.
+    1. Connect to the database server and ensure it's functioning.
+1. [Upgrade to the target server](#upgrading-to-the-target-server):
+    1. Stop both the source and the new server. 
+    1. Use pg_upgrade by specifying the source and target bin and data directories. 
+    1. Start the new database server. 
+    1. Connect to the encrypted database server and ensure the data was transferred. 
+1. [Clean up and delete the source server](#cleaning-up-after-upgrade):
+    1. Clean up the database and its statistics. 
+    1. Remove the source EDB Postgres Extended Server cluster with the script provided by pg_upgrade. 
+
+## Worked example
+
+This worked example upgrades an EDB Postgres Extended Server 15 database to EDB Postgres Extended Server 16. 
+
+!!!note 
+   You can perform major upgrades of EDB Postgres Extended Server instances in the same way you upgrade an EDB Postgres Advanced Server installation. If you need more information about the pg_upgrade utility, command line options, troubleshooting, and more, see [Upgrading an installation with pg_upgrade](/epas/latest/upgrading/04_upgrading_an_installation_with_pg_upgrade/).
+
+### Preparing your upgrade 
+
+Use [pg_dumpall](https://www.postgresql.org/docs/current/app-pg-dumpall.html), [pgBackRest](/supported-open-source/pgbackrest/), or [Barman](/supported-open-source/barman/) to create a backup of your source server. 
+
+### Installing the target EDB Postgres Extended Server version
+
+[Install EDB Postgres Extended Server](../installing/) version 16. Only install the packages. Don't perform any other configurations.
+
+### Creating a target server 
+
+If you don't want to create a new target instance but want to reuse an existing server with the target EDB Postgres Extended Server version, skip these steps and [ensure the target server is empty](/epas/latest/upgrading/04_upgrading_an_installation_with_pg_upgrade/03_upgrading_to_advanced_server/#step-2---empty-the-target-database). 
+
+1.  As postgres, create an empty directory for the new server: 
+  
+    ```
+    mkdir /var/lib/edb-pge/16/upgrade_target 
+    ```
+
+1.  As root, ensure the postgres user owns the directory:
+
+    ```
+    sudo chown enterprisedb /var/lib/edb-pge/16/upgrade_target
+    sudo chgrp /var/lib/edb-pge/16/upgrade_target
+    ```
+
+1.  As postgres, initialize the new server: 
+    
+    ```
+    /usr/lib/edb-pge/16/bin/initdb -D /var/lib/edb-pge/16/upgrade_target
+    ```
+
+    This command initializes a CONFIG directory with all configuration files for the encrypted server.
+
+1.  Before you start the cluster, ensure the new database runs on a different port from the source server. To alter the port, edit `postgresql.conf` by uncommenting the line with `#port` and changing the port number, for example, to 5432.
+
+1.  Start the target server: 
+
+    ```
+    /usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/TDE start
+    ```
```

Comment:
the data directory name is upgrade_target and not TDE 

 /usr/lib/edb-pge/16/bin/pg_ctl -D /var/lib/edb-pge/16/upgrade_target start